src/js/App.js

```diff
@@ -1,53 +1,181 @@
+        .then(([content, headers]) => {
+          checkIfModalShouldBeToggled(headers, toggleInnloggingsModal);
+          addInaktiveInnbokser(content);
+        })
+        .catch(() => {
+          setError();
+        });
+
+      Api.fetchInnloggingsstatus()
+        .then(([content]) => {
+          addInnloggingsstatus(content);
+        })
+        .catch(() => {
+          setError();
+        });
+    }, [],
+  );
+
+  useEffect(
+    () => {
+      Api.fetchOppfolging()
+        .then(([content, headers]) => {
+          checkIfModalShouldBeToggled(headers, toggleInnloggingsModal);
+          addOppfolging(content);
+        }).catch(() => {
+          setOppfolgingFailed();
+          setError();
+        });
+
+      Api.fetchMeldekort()
+        .then(([content, headers]) => {
+          checkIfModalShouldBeToggled(headers, toggleInnloggingsModal);
+          addMeldekort(content);
+        }).catch(() => {
+          setError();
+        });
+
+      Api.fetchPersonNavn()
+        .then(([content, headers]) => {
+          checkIfModalShouldBeToggled(headers, toggleInnloggingsModal);
+          addNavn(content);
+        }).catch(() => {
+          Api.fetchPersonIdent()
+            .then(([content]) => {
+              addIdent(content);
+              setError();
+            })
+            .catch(() => {
+              setError();
+            });
+        });
+
+      Api.fetchSaker()
+        .then(([content, headers]) => {
+          const { feilendeBaksystem } = content;
+          if (feilendeBaksystem.length > 0) {
+            addPaabegynteSoknader(content);
+            setError();
+          } else {
+            checkIfModalShouldBeToggled(headers, toggleInnloggingsModal);
+            addPaabegynteSoknader(content);
+          }
+        }).catch(() => {
+          setError();
+        });
+
+      Api.fetchMeldinger()
+        .then(([content, headers]) => {
+          checkIfModalShouldBeToggled(headers, toggleInnloggingsModal);
+          addMeldinger(content);
+        }).catch(() => {
+          setError();
+        });
+
+      Api.fetchSakstema()
+        .then(([content, headers]) => {
+          checkIfModalShouldBeToggled(headers, toggleInnloggingsModal);
+          addSakstema(content);
+        }).catch(() => {
+          setError();
+        });
+    }, [],
+  );
+
+  return (
+    <Router basename="/person">
+      <ScrollToTop />
+      <Switch>
+        <Route
+          path="/dittnav"
+          exact
+          component={Home}
+        />
         <Route
-          path="/dittnav/hendelser"
+          path="/dittnav/varslinger"
           exact
-          render={(props) => (
-            <HendelserTestSide {...props} />
-          )}
+          component={Varslinger}
         />
-      )}
-      <Route
-        path="/dittnav*"
-        exact
-        render={(props) => (
-          <RenderHome {...props} api={api} />
+        {Config.TEST_SIDE_FEATURE_TOGGLE && (
```

Comment:
Er dette feature-toggle for tidlinje-testing?

Reply:
Nei, men den kan godt brukes til det også.